src/Core/Entity/Image.php

```diff
@@ -13,8 +14,16 @@ class Image
 {
 
     /** Content TYPE */
-    const WEBP_CONTENT_TYPE = 'image/webp';
-    const JPEG_CONTENT_TYPE = 'image/jpeg';
+    const WEBP_MIME_TYPE = 'image/webp';
+    const JPEG_MIME_TYPE = 'image/jpeg';
+    const PNG_MIME_TYPE = 'image/png';
+
+    /** Extension output */
+    const EXT_AUTO = 'auto';
+    const EXT_PNG = 'png';
+    const EXT_WEBP = 'webp';
+    const EXT_JPEG = 'jpeg';
```

Comment:
This is not needed, jpeg extension should not be used on the web. The mime type is jpeg, the extension is jpg. Only obscure software and old macos9 gereate .jpeg files.